container/go/cmd/pusher/pusher.go

```diff
@@ -36,9 +37,28 @@ var (
 	src             = flag.String("src", "", "Path to the manifest.json when -format is legacy, path to the index.json when -format is oci or path to the image .tar file when -format is docker.")
 	format          = flag.String("format", "", "The format of the source image, (oci, legacy, or docker). The docker format should be a tarball of the image as generated by docker save.")
 	clientConfigDir = flag.String("client-config-dir", "", "The path to the directory where the client configuration files are located. Overiddes the value from DOCKER_CONFIG.")
+	layers          arrayFlags
 )
 
+const (
+	manifestPath = "manifest.json"
+	configPath   = "config.json"
+)
+
+// arrayFlags are defined for flags that may have multiple values.
+type arrayFlags []string
+
+func (f *arrayFlags) String() string {
+	return strings.Join(*f, "")
```

Comment:
Should the separator be an " "?

Reply:
Actually just returning fmt.Sprintf("%v", f) should work